### PR TITLE
add 4844 and set-batch-poster-manager contracts with Arb One and Nova values

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

File: src/gov-action-contracts/AIPs/AIP4844/ArbOneAIP4844Action.sol [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+/// @notice Upgrade ArbOne core contracts in preparation for 4844
+contract ArbOneAIP4844Action is AIP4844Action {
+    constructor()
+        AIP4844Action(
+            L1AddressRegistry(0xd514C2b3aaBDBfa10800B9C96dc1eB25427520A0), // L1 address registry for Arb One,
+            bytes32(""), // wasm module root  TODO
+            address(0), // new sequencer inbox impl TODO
+            address(0), // new challenge manager imp TODO
+            ProxyAdmin(0x5613AF0474EB9c528A34701A5b1662E3C8FA0678), // l1 gov proxy admin
+            address(0) // new one step proof  TODO
```
Comment (gzeoneth) [inline src/gov-action-contracts/AIPs/AIP4844/ArbOneAIP4844Action.sol:15]:
```suggestion
            0x31DA64D19Cd31A19CD09F4070366Fe2144792cf7, // new sequencer inbox impl
            0xE129b8Aa61dF65cBDbAE4345eE3fb40168DfD566, // new challenge manager impl
            ProxyAdmin(0x5613AF0474EB9c528A34701A5b1662E3C8FA0678), // l1 gov proxy admin
            0xC6E1E6dB03c3F475bC760FE20ed93401EC5c4F7e // new one step proof
```

File: src/gov-action-contracts/AIPs/AIP4844/NovaAIP4844Action.sol [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+/// @notice Upgrade Nova core contracts in preparation for 4844
+contract NovaAIP4844Action is AIP4844Action {
+    constructor()
+        AIP4844Action(
+            L1AddressRegistry(0x2F06643fc2CC18585Ae790b546388F0DE4Ec6635), // L1 address registry for Nova,
+            bytes32(""), // wasm module root  TODO
+            address(0), // new sequencer inbox impl TODO
+            address(0), // new challenge manager imp TODO
+            ProxyAdmin(0x5613AF0474EB9c528A34701A5b1662E3C8FA0678), // l1 gov proxy admin
+            address(0) // new one step proof  TODO
```
Comment (gzeoneth) [inline src/gov-action-contracts/AIPs/AIP4844/NovaAIP4844Action.sol:15]:
```suggestion
            0x31DA64D19Cd31A19CD09F4070366Fe2144792cf7, // new sequencer inbox impl
            0xE129b8Aa61dF65cBDbAE4345eE3fb40168DfD566, // new challenge manager impl
            ProxyAdmin(0x5613AF0474EB9c528A34701A5b1662E3C8FA0678), // l1 gov proxy admin
            0xC6E1E6dB03c3F475bC760FE20ed93401EC5c4F7e // new one step proof
```

**Review by gzeoneth — COMMENTED** [review]
https://github.com/OffchainLabs/nitro-contracts/pull/140

File: src/gov-action-contracts/AIPs/AIP4844/ArbOneAIP4844Action.sol [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+/// @notice Upgrade ArbOne core contracts in preparation for 4844
+contract ArbOneAIP4844Action is AIP4844Action {
+    constructor()
+        AIP4844Action(
+            L1AddressRegistry(0xd514C2b3aaBDBfa10800B9C96dc1eB25427520A0), // L1 address registry for Arb One,
+            bytes32(""), // wasm module root  TODO
+            0x31DA64D19Cd31A19CD09F4070366Fe2144792cf7, // new sequencer inbox impl
+            0xE129b8Aa61dF65cBDbAE4345eE3fb40168DfD566, // new challenge manager impl
+            ProxyAdmin(0x554723262467F125Ac9e1cDFa9Ce15cc53822dbD), // l1 gov proxy admin
```
Comment (DZGoldman) [inline src/gov-action-contracts/AIPs/AIP4844/ArbOneAIP4844Action.sol:14]:
```suggestion
            ProxyAdmin(0x554723262467F125Ac9e1cDFa9Ce15cc53822dbD), // l1 core contracts proxy admin for Arb One
```

File: src/gov-action-contracts/AIPs/AIP4844/NovaAIP4844Action.sol [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+/// @notice Upgrade Nova core contracts in preparation for 4844
+contract NovaAIP4844Action is AIP4844Action {
+    constructor()
+        AIP4844Action(
+            L1AddressRegistry(0x2F06643fc2CC18585Ae790b546388F0DE4Ec6635), // L1 address registry for Nova,
+            bytes32(""), // wasm module root  TODO
+            0x31DA64D19Cd31A19CD09F4070366Fe2144792cf7, // new sequencer inbox impl
+            0xE129b8Aa61dF65cBDbAE4345eE3fb40168DfD566, // new challenge manager impl
+            ProxyAdmin(0x71D78dC7cCC0e037e12de1E50f5470903ce37148), // l1 gov proxy admin
```
Comment (DZGoldman) [inline src/gov-action-contracts/AIPs/AIP4844/NovaAIP4844Action.sol:14]:
```suggestion
            ProxyAdmin(0x71D78dC7cCC0e037e12de1E50f5470903ce37148), //  L1 core contracts proxy admin for Nova

File: src/gov-action-contracts/AIPs/AIP4844/ArbOneAIP4844Action.sol [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+pragma solidity 0.8.16;
+
+import "./AIP4844Action.sol";
+
+/// @notice Upgrade ArbOne core contracts in preparation for 4844
+contract ArbOneAIP4844Action is AIP4844Action {
+    constructor()
+        AIP4844Action(
+            L1AddressRegistry(0xd514C2b3aaBDBfa10800B9C96dc1eB25427520A0), // L1 address registry for Arb One,
+            0x8b104a2e80ac6165dc58b9048de12f301d70b02a0ab51396c22b4b4b802a16a4, // arbos20 wasm module root
```
Comment (DZGoldman) [inline src/gov-action-contracts/AIPs/AIP4844/ArbOneAIP4844Action.sol:11]:
```suggestion
            0x8b104a2e80ac6165dc58b9048de12f301d70b02a0ab51396c22b4b4b802a16a4, // arbos20 wasm module root, built from https://github.com/OffchainLabs/nitro/commit/cf2eadfcc1039eca9594c4f71477a50f550d7749
```

**Review by gzeoneth — APPROVED** [review]

**Pull request merged**